src/mjolnir/graphbuilder.cc

```diff
@@ -732,10 +732,9 @@ void BuildTileSet(const std::string& ways_file,
               bike_network = w.bike_network();
             }
 
-            // Add edge info. Mean elevation is set to 1234 as a placeholder, set later if we have it.
             edge_info_offset =
                 graphtile.AddEdgeInfo(edge_pair.second, (*nodes[source]).graph_id,
-                                      (*nodes[target]).graph_id, w.way_id(), 1234, bike_network,
+                                      (*nodes[target]).graph_id, w.way_id(), 0, bike_network,
```

Comment:
I've not looked at this bug in a while but why not set to the kNoElevation value so that if it never gets set the bits of code that use it will know? Also we should consider what uses this value incase it would cause data compatibility issues

Reply:
yeah I think you're right. was a little quick last night. in the issue everyone said it should be 0, didn't challenge that much it seems. don't think there can be compatibility issues, I'll have a look again though and describe the flow a bit better